.gitignore

```diff
@@ -57,4 +57,4 @@ examples/int8/qat/qat
 examples/int8/training/vgg16/data/*
 examples/int8/datasets/data/*
 env/**/*
-bazel-Torch-TensorRT-Preview
\ No newline at end of file
+model_snapshot.txt
```

Comment:
can we call this model_manifest.txt or something?

Reply:
or torch_version.txt

Reply:
Changed it to model_manifest.json